src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements AreaOfFigure, InfoFigure {
+    //круг
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public double getRadius() {
+        return radius;
+    }
+
+    public void setRadius(double radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public double detAreaOfFigure() {
+        return Math.round(Math.PI * getRadius() * getRadius());
+    }
+
+    @Override
+    public String infoFigure() {
+        return ("Радиус2 = " + radius + '\n'
+                + "Площадь круга = " + detAreaOfFigure());
+    }
+
+    @Override
+    public String infoFromRigure() { // работает
```

Comment:
The method name `infoFromRigure` seems to be a typo. It should likely be `infoFromFigure` to match the naming convention and avoid confusion.